README.md

```diff
@@ -549,19 +549,6 @@ created above.
 
 #### 7. Configure Hasura
 
-1. Generate your JWT Secret key using [Hasura's tool -
-   https://hasura.io/jwt-config/](https://hasura.io/jwt-config/)
-   - Select Auth0
-   - Enter your Auth0 Domain
-1. Copy your key (the whole JSON object that is generated in the JWT Config
-   box) into your local `hasura/.env.local` file
-   - e.g. `HASURA_GRAPHQL_JWT_SECRET='your key goes in here'`
-   - **Don't forget the wrapping single quotes!**
-1. Uncomment the `HASURA_GRAPHQL_JWT_SECRET: ${HASURA_GRAPHQL_JWT_SECRET}`
-   line in `docker-compose.yaml`
-   - Don't forget to restart the `Hasura Console -- Local Development` task
-     in VSCode! (Again, if you get an error message about `version check: failed to get version from server: failed making version api call...`
-     try running the task again -- this could be due to a race condition.)
 1. Optionally: Copy your key into Hasura Cloud Environment configuration
```

Comment:
Come to think  of it, this bullet point should be deleted too - and in fact, the heading `#### 7. Configure Hasura` can then be deleted as well. Please update the numbering of subsequent headings (8, 9, 10)

Reply:
I have deleted this whole section and changed the order number accordingly. What's more, for Create Rules section, do we need to make some change to `Name it something like Hasura JWT` ?

Reply:
It's still required to follow the Hasura keys specification so I think the name of the rule remains appropriate.